### DISCUSSION TITLE
Is this expected behaviour for dynamic islands?

### DISCUSSION THREAD

**Discussion by WoodpeckerBaby** [qnblackcat/uYouPlus#724] [General]
![image](https://user-images.githubusercontent.com/14303318/204890308-6724d5ff-88d0-4742-8882-ac9ba1e3215d.jpeg)

Shouldn't it avoid cutting into important areas?

**Comment by yodaluca23** [comment]
I'm not able to reproduce this issue, are you on the latest release? Plus you know that there's a thing called scrolling, right?
![image](https://user-images.githubusercontent.com/67206487/204891087-fbdd0d2d-fe09-4178-994c-eed61ed9fdae.jpeg)

**Reply by WoodpeckerBaby** [reply]
You can replicate by using one of your fingers to tap on one of the normal videos. Then observe if the Dynamic Island eats into your video by a lot.

**Reply by yodaluca23** [reply]
> You can replicate by using one of your fingers to tap on one of the normal videos. Then observe if the Dynamic Island eats into your video by a lot. 

Still unable to replicate it, again, are you sure you're on the latest version? And have the right setting enabled?

![image](https://user-images.githubusercontent.com/67206487/205119671-cdb252db-f78f-4b7e-9098-d99b8a4335f6.png)
